### ISSUE TITLE
Hidden characters contributing to distance calculation

### ISSUE THREAD

**Issue by ells** [CDCgov/MicrobeTrace#189]
The TN93 calculator, in certain isolated cases, seems to be considering hidden characters (tabs, new lines, carriage returns, etc) as meaningful sequence data. 

Consider the following sequences that contain hidden characters, such as new lines:
```
>1				
TTTCTCTACTTTTTAACGAAAAAGAGGCAGATGACACTGACCATGAGCCCCAATAATCCCCCGGGGGCTTAGAGCCTCCCAGTGAAAAACATTTCCGCGAAACAGAAGTCTGAAAAGGTCAGGGCCCAGACT-AAGGCTCTGACGTCTCCCCCCGGGAGACAGCTCAGCACCGGCTCAGACTGGGCCCTGACGTGTCCCCCTGAAAGCGAATCATAAGCTCAGACCTCCGGGAAGCCACCGGGAACCACCCATTTCCTCCCCATATTTGTCAAGCCGTcTTCAGGCGTTGACGACAACCCCTCACCTCAAAAATTTTT--ATGGCACGCATACGGCTCAAT-AAAATAACAGGAGTCTATAAAAGCGTGGAGACAGTTCAGGAGGGGG--CTCGCTTCTTTCCCTCACGCGCCCGCCGCCCTACCTGAGGCCGCCATCCACGCCGGTTGAgTCG-CGTTCAGCCGCCTCCCGCCTGTGGTGCCTCCTGAACTGCGTCCGCTGTCTAGGTAAGTTTAGAGCTCAGGTCGAGACCGGGCCTTTGTCCGGCGCTCCCTTAGAGCCTACCTAGACCCAGCCGGCTCTCCACGCTTTGCCTGACCCTGCTTGCTCACCTCTACGTCTTTGTTTCGTTTTCTGT--TCTGCGCCG					
>2				
TTTCTCTACTTTTTAACGAAAAAGAGGCAGATGACACTGACCATGAGCCCCAAGTATCCCCCGGGGGCTCAGAGCCTCCCAGTGAAAAACATTTCCGCGAAACAGAAGTCTGAAAAGGTCAGGGCCCAGACT-GAGGCTCTGACGTCTCCCCCCGGGAGACAGCTCAGCACCGGCTCAGACTGGGCCCTGACGTGTCCCCCTGAAAACGAATCATAAGCTCAGGCCTCCGGGAAGCCACCAGGAACCACCCATTTCCTCCCCATATTTGTCAAGCCGTcTTCAGGCGTTGACGACAACCCCTCACCTCAAAAATTTTT--TTGGCACGCATACGGCTCAAT-AAAATAACAGGAGTCTATAAAAGCGTGGAGACAGTTCAGGAGGGGG--CTCGCTTCTTTCCCTCACGCGCCCGCCGCCCTACCTGAGGCCGCCATCCACGCCGGTTGAgTCG-CGTTCTGCCGCCTCCCGCCTGTGGTGCCTCCTGAACTGCGTCCGCCGTCTAGGTAAGTTTAGAGCTCAGGTCGAGACCGGGCCTTTGTCCGGCGCTCCCTTAGAGCCTACCTAGACTCAGCCGGCTCTCCACGCTTTGCCTGACCCTGTTTGCTCACCTCTACGTCTTTGTTTCGTTTTCTGT--TCTGCGCCG					
>3				
TTTCTCTACTTTTTAACGAAAAGGAGGCAGATGACACTGACCATGAGCCCCAAGTATCCCCC-GGGGCTTAGAGCCTCCCAGTGAAAAACATTTCCGCGAAACAGAAGTCTGAAAAGGTCAGGGCCCAGACT-GAGGCTCTGACGTCTCCCCCCGGGAGACAGCTCAGCACCGGCTCAGACT-GGCCCTGACGTGTCCCCCTGAAAACGAATCATAAGCTCAGGCCTCC-GGAAGCCACCGGGAACCACCCATTTCCTCCCCATATTTGTCAAGCCGTcTTCAGGCGTTGACGACAACCTCTCACCTCAAAAATTTTT--ATGGCACGCATACGGCTCAAT-AAAATAACAGGAGTCTATAAAAGCGTGGAGACAGTTCAGGAGGGGG--CTCGCTTCTTTCCCTCACGCGCCCGCCGCCCTACCTGAGGCCGCCATCCACGCCGGTTGAgTCG-CGTTCTGCCGCCTCCCGCCTGTGGTGCCTCCTGAACTGCGTCCGCCGTCTAGGTAAGTTTAGAGCTCAGGTCGAGACCGGGCCTTTGTCCGGCGCTCCCTTAGAGCCTACCTAGACTCAGCCGGCTCTCCACGCTTTGCCTGACCCTGTTTGCTCACCTCTACGTCTTTGTTTCGTTTTCTGT--TCTGCGCCG					
>4				
TTTCTCTACTTTTTAACGAAAAGGAGGCAGATGACACTGACCATGAGCCCCAAATATCCCCCGGGGGCTTAGAGCCTCCCAGTGAAAAACATTTCCGCGAAACAGAAGTCTGAAAAGGTCAGGGCCCAGACT-GAGGCTCTGACGTCTCCCCCCGGGAGACAGCTCAGCACCGGCTCAGACTGGGCCCTGACGTGTCCCCCTGAAAACGAATCACAAGCTCAGGCCTCCGGGAAGCCACCAGGAACCACCCATTTCCTCCCCATATTTGTCAAGCCGTcTTCAGGCGTTGACGACAACCCCTCACCTCAAAAATTTTT--ATGGCACGCATACGGCTCAAT-AAAATAACAGGAGTCTATAAAAGCGTGGAGACAGTTCAGGAGGGGG--CTCGCTTCTTTCCCTCACGCGCCCGCCGCCCTACCTGAGGCCGCCATCCACGCCGGTTGAgTCG-CGTTCTGCCGCCTCCCGCCTGTGGTGCCTCCTGAACTGCGTCCGCCGTCTAGGTAAGTTTAGAGCTCAGGTCGAGACCGGGCCTTTGTCCGGCGCTCCCTTAGAGCCTACCTAGACTCAGCCGGCTCTCCACGCTTTGCCTGACCCTGTTTGCTCACCTCTACGTCTTTGTTTCGTTTTCTGT--TCTGCGCCG					
>5				
TTTCTCTACTTTTTAACGAAAAAGAGGCAGATGACACTGACCATGAGCCCCAAGTATCCCCCGGGGGCTTAGAGCCTCCCAGTGAAAAACATTTCCGCGAAACAGAAGTCTGAAAAGGTCAGGGCCCAGACT-GAGGCTCTGACGTCTCCCCCCGGGAGACAGCTCAGCACCGGCTCAGACTGGGCCCTGACGTGTCCCCCTGAAAACGAATCATAAGCTCAGGCCTCCGGGAAGCCACCAGGAACCACCCATTTCCTCCCCATATTTGTCAAGCCGTcTTCAGGCGTTGACGACAACCCCTCACCTCAAAAATTTTT--ATGGCACGCATACGGCTCAAT-AAAATAACAGGAGTCTATAAAAGCGTGGAGACAGTTCAGGAGGGGG--CTCGCTTCTTTCCCTCACGCGCCCGCCGCCCTACCTGAGGCCGCCATCCACGCCGGTTGAgTCG-CGTTCTGCCGCCTCCCGCCTGTGGTGCCTCCTGAACTGCGTCCGCCGTCTAGGTAAGTTTAGAGCTCAGGTCGAGACCGGGCCTTTGTCCGGCGCTCCCTTAGAGCCTACCTAGACTCAGCCGGCTCTCCACGCTTTGCCTGACCCTGTTTGCTCACCTCTACGTCTTTGTTTCGTTTTCTGT--TCTGCGCCG					
>6				
TTTCTCTACTTTTTAACGAAAAAGAGGCAGATGACACTGACCATGAGCCCCAAGTATCCCCCGGGGGCTTAGAGCCTCCCAGTGAAAAACATTTCCGCGAAACAGAAGTCTGAAAAGGTCAGGGCCCAGACT-GAGGCTCTGACGTCTCCCCCCGGGAGACAGCTCAGCACCGGCTCAGACTGGGCCCTGACGTGTCCCCCTGAAAACGAATCATAAGCTCAGGCCTCCGGGAAGCCACCAGGAACCACCCATTTCCTCCCCATATTTGTCAAGCCGTcTTCAGGCGTTGACGACAACCCCTCACCTCAAAAATTTTT--ATGGCACGCATACGGCTCAAT-AAAATAACAGGAGTCTATAAAAGCGTGGAGACAGTTCAGGAGGGGG--CTCGCTTCTTTCCCTCACGCGCCCGCCGCCCTACCTGAGGCCGCCATCCACGCC-GTTGAgTCGCCGTTCTGCCGCCTCCCGCCTGTGGTGCCTCCTGAACTGCCTCCGCCGTCTAGGTAAGTTTAGAGCTCAGGTCGAGACCGGGCCTTTGTCCGGCGCTCCCTTAGAGCCTACCTAGACTCAGCCGGCTCTCCACGCTTTGCCTGACCCTGTTTGCTCACCTCTACGTCTTTGTTTCGTTTTCTGT--TCTGCGCCG					
>7				
TTTCTCTACTTTTTAACGAAAAGGAGGCAGATGACACTGACCATGAGCCCCAAGTATCCCCCGGGGGCTTAGAGCCTCCCAGTGAAAAACATTTCCGCGAAACAGAAATCTGAAAAGGTCAGGGCCCAGACT-GAGGCTCTGACGTCTCCCCCCGGGAGACAGCTCAGCACCGGCTCAGACTGGGCCCTGACGTGTCCCCCTGAAAACGAATCACAAGCTCAGGCCTCCGGGAAGCCACCGGGAACCACCCATTTCCTCCCCATATTTGTCAAGCCGTcTTCAGGCGTTGACGACAACCCCTCACCTCAAAAATTTTT--ATGGCACGCATACGGCTCAAT-AAAATAACAGGAGTCTATAAAAGCGTGGAGACAGTTCAGGAGGGGG--CTCGCTTCTTTCCCTCACGCGCCCGCCGCCCTACCTGAGGCCGCCATCCACGCCGGTTGAgTCG-CGTTCTGCCGCCTCCCGCCTGTGGTGCCTCCTGAACTGCGTCCGCCGTCTAGGTAAGTTTAGAGCTCAGGTCGAGACCGGGCCTTTGTCCGGCGCTCCCTTAGAGCCTACCTAGACTCAGCCGGCTCTCCACGCTTTGCCTGACCCTGTTTGCTCACCTCTACGTCTTTGTTTCGTTTTCTGT--TCTGCGCCG					
>8					
TTTCTCTACTTTTTAACGAAAAGGAGGCAGATGACACTGACCATGAGCCCCAAATATCCCCCGGGGGCTTAGAGCCTCCCAGTGAAAAACATTTCCGCGAAACAGAAGTCTGAAAAGGTCAGGGCCCAGACTAAAGGCTCTGACGTCTCCCCCCGGGAGACAGCTCAGCACCGGCTCAGACTGGGCCCTGACGTGTCCCCCTGAAAGCGAATCATAAGCTCAGACCTCCGGGAAGCCACCGGGAACCACCCATTTCCTCCCCATATTTGTCAAGCCGTcTTCAGGCGTTGACGACAACCCCTCACCCCAAAAATTTTT--ATGGCACGCATACGGCTCAAT-AAAATAACAGGAGTCTATAAAAGCGTGGAGACAGTTCAGGAGGGGG--CTCGCTTCTTTCCCTCACGCGCCCGCCGCCCTACCTGAGGCCGCCATCCACGCCGGTTGAgTCG-CGTTCTGCCGCCTCCCGCCTGTGGTGCCTCCTGAACTGCGTCCGCTGTCTAGGTAAGTTTAGAGCTCAGGTCGAGACCGGGCCTTTGTCCGGCGCTCCCTTAGAGCCTACCTAGACCCAGCCGGCTCTCCACGCTTTGCCTGACCCTGCTTGCTCACCTCTACGTCTTTGTTTCGTTTTCTGT--TCTGCGCCG					
>9				
TTTCTCTACTTTTTAACGAAAAGGAGGCAGATGACACTGACCATGAGCCCCAAGTATCCCCCGGGGGCTTAGAGCCTCCCAGTGAAAAACATTTCCGCGAAACAGAAGTCTGAAAAGGTCAGGGCCCAGACT-GAGGCTCTGACGTCTCCCCCCGGGAGACAGCTCAGCACCGGCTCAGACTGGGCCCTGACGTGTCCCCCTGAAAACGAATCATAAGCTCAGGCCTCCGGGAAGCCACCGGGAACCACCCATTTCCTCCCCATATTTGTCAAGCCGTcTTCAGGCATTGACGACAACCCCTCACCTCAAAAATTTTT--ATGGCACGCATACGGCTCAAT-AAAATAACAGGAGTCTATAAAAGCGTGGAGACAGTTCAGGAGGGGG--CTCGCTTCTTTCCCTCACGCGCCCGCCGCCCTACCTGAGGCCGCCATCCACGCCGGTTGAgTCG-CGTTCTGCCGCCTCCCGCCTGTGGTGCCTCCTGAACTGCGTCCGCCGTCTAGGTAAGTTTAGAGCTCAGGTCGAGACCGGGCCTTTGTCCGGCGCTCCCTTAGAGCCTACCTAGACTCAGCCGGCTCTCCACGCTTTGCCTGACCCTGTTTGCTCACCTCTACGTCTTTGTTTCGTTTTCTGT--TCTGCGCCG					
>10					
TTTCTCTACTTTTTAACGAAAAAGAGGCAGATGACACTGACCATGAGCCCCAAGTATCCCCCGGGGGCTTAGAGCCTCCCAGTGAAAAACATTTCCGCGAAACAGAAGTCTGAAAAGGTCAGGGCCCAGACT-GAGGCTCTGACGTCTCCCCCCGGGAGACAGCTCAGCACCGGCTCAGACTGGGCCCTGACGTGTCCCCCTGAAAACGAATCATAAGCTCAGGCCTCCGGGAAGCCACCGGGAACCACCCATTTCCTCCCCATATTTGTCAAGCCGTcTTCAGGCATTGACGACAACCCCTCACCTCAAAAATTTTT--ATGGCACGCATACGGCTCAAT-AAAATAACAGGAGTCTATAAAAGCGTGGAGACAGTTCAGGAGGGGG--CTCGCTTCTTTCCCTCACGCGCCCGCCGCCCTACCTGAGGCCGCCATCCACGCCGGTTGAgTCG-CGTTCTGCCGCCTCCCGCCTGTGGTGCCTCCTGAACTGCGTCCGCCGTCTAGGTAAGTTTAGAGCTCAGGTCGAGACCGGGCCTTTGTCCGGCGCTCCCTTAGAGCCTACCTAGACTCAGCCGGCTCTCCACGCTTTGCCTGACCCTGTTTGCTCACCTCTACGTCTTTGTTTCGTTTTCTGT--TCTGCGCCG					
```
This results in the following network diagram in MicrobeTrace:
![image](https://user-images.githubusercontent.com/1420108/73103024-1af38a80-3ec1-11ea-8a59-b7c6667db99b.png)

This contains sequences without new lines:
```
>1				
TTTCTCTACTTTTTAACGAAAAAGAGGCAGATGACACTGACCATGAGCCCCAATAATCCCCCGGGGGCTTAGAGCCTCCCAGTGAAAAACATTTCCGCGAAACAGAAGTCTGAAAAGGTCAGGGCCCAGACT-AAGGCTCTGACGTCTCCCCCCGGGAGACAGCTCAGCACCGGCTCAGACTGGGCCCTGACGTGTCCCCCTGAAAGCGAATCATAAGCTCAGACCTCCGGGAAGCCACCGGGAACCACCCATTTCCTCCCCATATTTGTCAAGCCGTcTTCAGGCGTTGACGACAACCCCTCACCTCAAAAATTTTT--ATGGCACGCATACGGCTCAAT-AAAATAACAGGAGTCTATAAAAGCGTGGAGACAGTTCAGGAGGGGG--CTCGCTTCTTTCCCTCACGCGCCCGCCGCCCTACCTGAGGCCGCCATCCACGCCGGTTGAgTCG-CGTTCAGCCGCCTCCCGCCTGTGGTGCCTCCTGAACTGCGTCCGCTGTCTAGGTAAGTTTAGAGCTCAGGTCGAGACCGGGCCTTTGTCCGGCGCTCCCTTAGAGCCTACCTAGACCCAGCCGGCTCTCCACGCTTTGCCTGACCCTGCTTGCTCACCTCTACGTCTTTGTTTCGTTTTCTGT--TCTGCGCCG
>2				
TTTCTCTACTTTTTAACGAAAAAGAGGCAGATGACACTGACCATGAGCCCCAAGTATCCCCCGGGGGCTCAGAGCCTCCCAGTGAAAAACATTTCCGCGAAACAGAAGTCTGAAAAGGTCAGGGCCCAGACT-GAGGCTCTGACGTCTCCCCCCGGGAGACAGCTCAGCACCGGCTCAGACTGGGCCCTGACGTGTCCCCCTGAAAACGAATCATAAGCTCAGGCCTCCGGGAAGCCACCAGGAACCACCCATTTCCTCCCCATATTTGTCAAGCCGTcTTCAGGCGTTGACGACAACCCCTCACCTCAAAAATTTTT--TTGGCACGCATACGGCTCAAT-AAAATAACAGGAGTCTATAAAAGCGTGGAGACAGTTCAGGAGGGGG--CTCGCTTCTTTCCCTCACGCGCCCGCCGCCCTACCTGAGGCCGCCATCCACGCCGGTTGAgTCG-CGTTCTGCCGCCTCCCGCCTGTGGTGCCTCCTGAACTGCGTCCGCCGTCTAGGTAAGTTTAGAGCTCAGGTCGAGACCGGGCCTTTGTCCGGCGCTCCCTTAGAGCCTACCTAGACTCAGCCGGCTCTCCACGCTTTGCCTGACCCTGTTTGCTCACCTCTACGTCTTTGTTTCGTTTTCTGT--TCTGCGCCG
>3				
TTTCTCTACTTTTTAACGAAAAGGAGGCAGATGACACTGACCATGAGCCCCAAGTATCCCCC-GGGGCTTAGAGCCTCCCAGTGAAAAACATTTCCGCGAAACAGAAGTCTGAAAAGGTCAGGGCCCAGACT-GAGGCTCTGACGTCTCCCCCCGGGAGACAGCTCAGCACCGGCTCAGACT-GGCCCTGACGTGTCCCCCTGAAAACGAATCATAAGCTCAGGCCTCC-GGAAGCCACCGGGAACCACCCATTTCCTCCCCATATTTGTCAAGCCGTcTTCAGGCGTTGACGACAACCTCTCACCTCAAAAATTTTT--ATGGCACGCATACGGCTCAAT-AAAATAACAGGAGTCTATAAAAGCGTGGAGACAGTTCAGGAGGGGG--CTCGCTTCTTTCCCTCACGCGCCCGCCGCCCTACCTGAGGCCGCCATCCACGCCGGTTGAgTCG-CGTTCTGCCGCCTCCCGCCTGTGGTGCCTCCTGAACTGCGTCCGCCGTCTAGGTAAGTTTAGAGCTCAGGTCGAGACCGGGCCTTTGTCCGGCGCTCCCTTAGAGCCTACCTAGACTCAGCCGGCTCTCCACGCTTTGCCTGACCCTGTTTGCTCACCTCTACGTCTTTGTTTCGTTTTCTGT--TCTGCGCCG
>4				
TTTCTCTACTTTTTAACGAAAAGGAGGCAGATGACACTGACCATGAGCCCCAAATATCCCCCGGGGGCTTAGAGCCTCCCAGTGAAAAACATTTCCGCGAAACAGAAGTCTGAAAAGGTCAGGGCCCAGACT-GAGGCTCTGACGTCTCCCCCCGGGAGACAGCTCAGCACCGGCTCAGACTGGGCCCTGACGTGTCCCCCTGAAAACGAATCACAAGCTCAGGCCTCCGGGAAGCCACCAGGAACCACCCATTTCCTCCCCATATTTGTCAAGCCGTcTTCAGGCGTTGACGACAACCCCTCACCTCAAAAATTTTT--ATGGCACGCATACGGCTCAAT-AAAATAACAGGAGTCTATAAAAGCGTGGAGACAGTTCAGGAGGGGG--CTCGCTTCTTTCCCTCACGCGCCCGCCGCCCTACCTGAGGCCGCCATCCACGCCGGTTGAgTCG-CGTTCTGCCGCCTCCCGCCTGTGGTGCCTCCTGAACTGCGTCCGCCGTCTAGGTAAGTTTAGAGCTCAGGTCGAGACCGGGCCTTTGTCCGGCGCTCCCTTAGAGCCTACCTAGACTCAGCCGGCTCTCCACGCTTTGCCTGACCCTGTTTGCTCACCTCTACGTCTTTGTTTCGTTTTCTGT--TCTGCGCCG
>5				
TTTCTCTACTTTTTAACGAAAAAGAGGCAGATGACACTGACCATGAGCCCCAAGTATCCCCCGGGGGCTTAGAGCCTCCCAGTGAAAAACATTTCCGCGAAACAGAAGTCTGAAAAGGTCAGGGCCCAGACT-GAGGCTCTGACGTCTCCCCCCGGGAGACAGCTCAGCACCGGCTCAGACTGGGCCCTGACGTGTCCCCCTGAAAACGAATCATAAGCTCAGGCCTCCGGGAAGCCACCAGGAACCACCCATTTCCTCCCCATATTTGTCAAGCCGTcTTCAGGCGTTGACGACAACCCCTCACCTCAAAAATTTTT--ATGGCACGCATACGGCTCAAT-AAAATAACAGGAGTCTATAAAAGCGTGGAGACAGTTCAGGAGGGGG--CTCGCTTCTTTCCCTCACGCGCCCGCCGCCCTACCTGAGGCCGCCATCCACGCCGGTTGAgTCG-CGTTCTGCCGCCTCCCGCCTGTGGTGCCTCCTGAACTGCGTCCGCCGTCTAGGTAAGTTTAGAGCTCAGGTCGAGACCGGGCCTTTGTCCGGCGCTCCCTTAGAGCCTACCTAGACTCAGCCGGCTCTCCACGCTTTGCCTGACCCTGTTTGCTCACCTCTACGTCTTTGTTTCGTTTTCTGT--TCTGCGCCG
>6				
TTTCTCTACTTTTTAACGAAAAAGAGGCAGATGACACTGACCATGAGCCCCAAGTATCCCCCGGGGGCTTAGAGCCTCCCAGTGAAAAACATTTCCGCGAAACAGAAGTCTGAAAAGGTCAGGGCCCAGACT-GAGGCTCTGACGTCTCCCCCCGGGAGACAGCTCAGCACCGGCTCAGACTGGGCCCTGACGTGTCCCCCTGAAAACGAATCATAAGCTCAGGCCTCCGGGAAGCCACCAGGAACCACCCATTTCCTCCCCATATTTGTCAAGCCGTcTTCAGGCGTTGACGACAACCCCTCACCTCAAAAATTTTT--ATGGCACGCATACGGCTCAAT-AAAATAACAGGAGTCTATAAAAGCGTGGAGACAGTTCAGGAGGGGG--CTCGCTTCTTTCCCTCACGCGCCCGCCGCCCTACCTGAGGCCGCCATCCACGCC-GTTGAgTCGCCGTTCTGCCGCCTCCCGCCTGTGGTGCCTCCTGAACTGCCTCCGCCGTCTAGGTAAGTTTAGAGCTCAGGTCGAGACCGGGCCTTTGTCCGGCGCTCCCTTAGAGCCTACCTAGACTCAGCCGGCTCTCCACGCTTTGCCTGACCCTGTTTGCTCACCTCTACGTCTTTGTTTCGTTTTCTGT--TCTGCGCCG
>7				
TTTCTCTACTTTTTAACGAAAAGGAGGCAGATGACACTGACCATGAGCCCCAAGTATCCCCCGGGGGCTTAGAGCCTCCCAGTGAAAAACATTTCCGCGAAACAGAAATCTGAAAAGGTCAGGGCCCAGACT-GAGGCTCTGACGTCTCCCCCCGGGAGACAGCTCAGCACCGGCTCAGACTGGGCCCTGACGTGTCCCCCTGAAAACGAATCACAAGCTCAGGCCTCCGGGAAGCCACCGGGAACCACCCATTTCCTCCCCATATTTGTCAAGCCGTcTTCAGGCGTTGACGACAACCCCTCACCTCAAAAATTTTT--ATGGCACGCATACGGCTCAAT-AAAATAACAGGAGTCTATAAAAGCGTGGAGACAGTTCAGGAGGGGG--CTCGCTTCTTTCCCTCACGCGCCCGCCGCCCTACCTGAGGCCGCCATCCACGCCGGTTGAgTCG-CGTTCTGCCGCCTCCCGCCTGTGGTGCCTCCTGAACTGCGTCCGCCGTCTAGGTAAGTTTAGAGCTCAGGTCGAGACCGGGCCTTTGTCCGGCGCTCCCTTAGAGCCTACCTAGACTCAGCCGGCTCTCCACGCTTTGCCTGACCCTGTTTGCTCACCTCTACGTCTTTGTTTCGTTTTCTGT--TCTGCGCCG
>8
TTTCTCTACTTTTTAACGAAAAGGAGGCAGATGACACTGACCATGAGCCCCAAATATCCCCCGGGGGCTTAGAGCCTCCCAGTGAAAAACATTTCCGCGAAACAGAAGTCTGAAAAGGTCAGGGCCCAGACTAAAGGCTCTGACGTCTCCCCCCGGGAGACAGCTCAGCACCGGCTCAGACTGGGCCCTGACGTGTCCCCCTGAAAGCGAATCATAAGCTCAGACCTCCGGGAAGCCACCGGGAACCACCCATTTCCTCCCCATATTTGTCAAGCCGTcTTCAGGCGTTGACGACAACCCCTCACCCCAAAAATTTTT--ATGGCACGCATACGGCTCAAT-AAAATAACAGGAGTCTATAAAAGCGTGGAGACAGTTCAGGAGGGGG--CTCGCTTCTTTCCCTCACGCGCCCGCCGCCCTACCTGAGGCCGCCATCCACGCCGGTTGAgTCG-CGTTCTGCCGCCTCCCGCCTGTGGTGCCTCCTGAACTGCGTCCGCTGTCTAGGTAAGTTTAGAGCTCAGGTCGAGACCGGGCCTTTGTCCGGCGCTCCCTTAGAGCCTACCTAGACCCAGCCGGCTCTCCACGCTTTGCCTGACCCTGCTTGCTCACCTCTACGTCTTTGTTTCGTTTTCTGT--TCTGCGCCG
>9				
TTTCTCTACTTTTTAACGAAAAGGAGGCAGATGACACTGACCATGAGCCCCAAGTATCCCCCGGGGGCTTAGAGCCTCCCAGTGAAAAACATTTCCGCGAAACAGAAGTCTGAAAAGGTCAGGGCCCAGACT-GAGGCTCTGACGTCTCCCCCCGGGAGACAGCTCAGCACCGGCTCAGACTGGGCCCTGACGTGTCCCCCTGAAAACGAATCATAAGCTCAGGCCTCCGGGAAGCCACCGGGAACCACCCATTTCCTCCCCATATTTGTCAAGCCGTcTTCAGGCATTGACGACAACCCCTCACCTCAAAAATTTTT--ATGGCACGCATACGGCTCAAT-AAAATAACAGGAGTCTATAAAAGCGTGGAGACAGTTCAGGAGGGGG--CTCGCTTCTTTCCCTCACGCGCCCGCCGCCCTACCTGAGGCCGCCATCCACGCCGGTTGAgTCG-CGTTCTGCCGCCTCCCGCCTGTGGTGCCTCCTGAACTGCGTCCGCCGTCTAGGTAAGTTTAGAGCTCAGGTCGAGACCGGGCCTTTGTCCGGCGCTCCCTTAGAGCCTACCTAGACTCAGCCGGCTCTCCACGCTTTGCCTGACCCTGTTTGCTCACCTCTACGTCTTTGTTTCGTTTTCTGT--TCTGCGCCG
>10
TTTCTCTACTTTTTAACGAAAAAGAGGCAGATGACACTGACCATGAGCCCCAAGTATCCCCCGGGGGCTTAGAGCCTCCCAGTGAAAAACATTTCCGCGAAACAGAAGTCTGAAAAGGTCAGGGCCCAGACT-GAGGCTCTGACGTCTCCCCCCGGGAGACAGCTCAGCACCGGCTCAGACTGGGCCCTGACGTGTCCCCCTGAAAACGAATCATAAGCTCAGGCCTCCGGGAAGCCACCGGGAACCACCCATTTCCTCCCCATATTTGTCAAGCCGTcTTCAGGCATTGACGACAACCCCTCACCTCAAAAATTTTT--ATGGCACGCATACGGCTCAAT-AAAATAACAGGAGTCTATAAAAGCGTGGAGACAGTTCAGGAGGGGG--CTCGCTTCTTTCCCTCACGCGCCCGCCGCCCTACCTGAGGCCGCCATCCACGCCGGTTGAgTCG-CGTTCTGCCGCCTCCCGCCTGTGGTGCCTCCTGAACTGCGTCCGCCGTCTAGGTAAGTTTAGAGCTCAGGTCGAGACCGGGCCTTTGTCCGGCGCTCCCTTAGAGCCTACCTAGACTCAGCCGGCTCTCCACGCTTTGCCTGACCCTGTTTGCTCACCTCTACGTCTTTGTTTCGTTTTCTGT--TCTGCGCCG
```
This second set of sequences results in the following diagram:
![image](https://user-images.githubusercontent.com/1420108/73102992-0c0cd800-3ec1-11ea-8571-20a2baa83553.png)

I have confirmed that copy & pasting the above sequences into a fasta file DOES preserve the hidden characters that cause the problem. Therefore, these should be sufficient in replicating the issue.

**Comment by jaywokim** [comment]
noticed extra spaces in source files for FAS:
in ID part
![Capture](https://user-images.githubusercontent.com/55855925/73421846-23025e80-42f5-11ea-97a0-00502365a932.PNG)
in SEQ part
![Capture](https://user-images.githubusercontent.com/55855925/73421924-5644ed80-42f5-11ea-9b0d-67cb6efafd25.PNG)
after trimming extra space I get the first image:
![Capture](https://user-images.githubusercontent.com/55855925/73421962-796f9d00-42f5-11ea-9bdb-d2aa0e67f8c4.PNG)

**Comment by jaywokim** [comment]
resolved with same update for issue #190
